drivers/lcd/Kconfig

```diff
@@ -619,15 +625,15 @@ if LCD_ST7789
 		---help---
 			Specifies the Y offset of the LCD.
 
-	config LCD_ST7789_BPP
+	config LCD_ST77XX_BPP
```

Comment:
I'm concerned about having two `config` options with the same identifier.
Have you tested enabling both `LCD_ST7735` and `LCD_ST7789`?
This is a valid scenario and I believe will probably result in a crash in `Kconfig`.

Reply:
Hi @gustavonihei I´ve just tried enabling both ST7735 and ST7789 and it configured and compiled NuttX without any problems. But when I run the fb example application it took the resolution of ST7735 instead of ST7789 because of those #ifdefs https://github.com/michallenc/incubator-nuttx/blob/st77xx/drivers/lcd/st77xx.c#L97.

I think the changes I did would not allow to use both 7735 and 7789 at the same time (because of resolution and also some other stuffs that are under ifdefs). It is a question whether we want to keep that option of using both controllers (I don´t know how much are two different displays used, haven´t seen much of those options myself) or to keep those controllers under one file.

Reply:
@michallenc maybe you can define the CONFIG_LCD_ST77XX as main option and after enabling it the user can select (using choice) his model as ST7735 or ST7789

Reply:
> @michallenc maybe you can define the CONFIG_LCD_ST77XX as main option and after enabling it the user can select (using choice) his model as ST7735 or ST7789

Hi @acassis, sorry for the late reply. I´ve already started with those changes you proposed, haven´t pushed them to GIT yet. It still does not solve the problem that it would not allow the user to use both ST7735 and ST7789 (or the driver mentiopned by @PeterBee97 below) drivers together (although I don´t think more than one display is usually used by the user, so it might not be such a problem).